.github/workflows/test-analysis-failure.yml

```diff
@@ -10,10 +10,10 @@ on:
 jobs:
   call-test-analysis-failure-nuget:
     name: Test Analysis Failure - NuGet PackageReference
-    if: ${{ false }}
+    if: false
```

Comment:
This still seems like a leftover (we don't need a no-op step).

Reply:
Yes, fixed!